core/src/main/scala/cats/Traverse.scala

```diff
@@ -30,6 +30,12 @@ import simulacrum.typeclass
     U.TC.traverse(fa)(a => U.subst(f(a)))(this)
 
   /**
+   * A traverse followed by flattening the inner result.
+   */
+  def traverseM[G[_], A, B](fa: F[A])(f: A => G[F[B]])(implicit G: Applicative[G], F: FlatMap[F]): G[F[B]] =
+    G.map(traverse(fa)(f))(F.flatten)
```

Comment:
Oh I totally did not see the `FlatMap` requirement.. the size of my screen stopped right at `Applicative` so I assumed that was all there was. I am good with `traverseM` in that case